nautobot/utilities/testing/views.py

```diff
@@ -580,6 +580,8 @@ def test_list_objects_filtered(self):
             content = extract_page_body(response.content.decode(response.charset))
             # TODO: it'd make test failures more readable if we strip the page headers/footers from the content
             if hasattr(self.model, "name"):
+                # TODO: this is fragile as we move toward more autogenerated test fixtures,
+                #       as "instance.name" may appear coincidentally in other page text if we're not careful.
```

Comment:
Since I'm currently in the midst of invententorying `TODO` comments I'm a bit sensitive to this right now. Would the preceding comment about stripping headers/footers help improve this test and make it be less fragile? Is there something immediately actionable otherwise?

Reply:
Nothing immediately actionable - in an earlier version of this PR I changed things to look for the table rows instead of the instance names, but that broke for the Job table/list view since it renders the table rows differently, so I reverted things back to this baseline. Stripping headers/footers would probably help but it's still a risk - the main mitigation is setting up factories to generate names that are not likely to appear by coincidence in the page text, such as by using `.capitalize()` or `.upper()` on random-word names so that they don't accidentally match in the middle of some other word.

Do you want me to remove the `TODO` portion of the comment and just leave it as an informational note?

Reply:
Opened https://github.com/nautobot/nautobot/issues/2580 as a housekeeping item. I'll add it to the comment here as a reference.